### DISCUSSION TITLE
🥰 [FEATURE]

### DISCUSSION THREAD

**Discussion by ChangeZhang** [antvis/G2Plot#3589] [Q&A] [answered]
### 🚑 Any additional [like screenshots]
希望加一些基本地图和地理位置的demo示例

**Comment by hustcc** [comment, marked as answer]
地图相关，建议使用 [L7](https://l7.antv.antgroup.com/) 技术栈相关的内容。

- 直接使用 L7 地图引擎绘制灵活的地图
- 使用 [L7Plot](https://l7plot.antv.antgroup.com/) 绘制封装好的地图
